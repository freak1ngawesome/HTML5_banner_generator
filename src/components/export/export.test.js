import React from 'react';
import Export from './export';
import renderer from 'react-test-renderer';
import store from '../../store';
import { shallow } from 'enzyme';

describe('Testing <Export/>', () => {
  const exp = renderer.create(<Export store={store}/>);

  it('Component has rendered correctly',() => {
    expect(exp.toJSON()).toMatchSnapshot();
  });


});
