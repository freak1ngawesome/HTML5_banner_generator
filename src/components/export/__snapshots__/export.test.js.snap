// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Export/> Component has rendered correctly 1`] = `
Array [
  <section
    className="export"
  >
    <button
      className="export__btn"
      onClick={[Function]}
    >
      Save as .png
    </button>
    <button
      className="export__btn"
      onClick={[Function]}
    >
      Copy as HTML5 string
    </button>
    <button
      className="export__btn"
      onClick={[Function]}
    >
      Copy settings
    </button>
  </section>,
  <p
    className="notification"
  />,
]
`;
